app/services/idv/steps/ipp/ssn_step.rb

```diff
@@ -20,6 +25,11 @@ def extra_view_variables
         def form_submit
           Idv::SsnFormatForm.new(current_user).submit(permit(:ssn))
         end
+
+        def threatmetrix_session_id
+          return nil if !IdentityConfig.store.proofing_device_profiling_collecting_enabled
```

Comment:
Do we need to check `IdentityConfig.store.proofing_device_profiling_collecting_enabled` twice (here and in the `call` function)?